framework/validators/InlineValidator.php

```diff
@@ -19,6 +19,15 @@
  * where `$attribute` refers to the name of the attribute being validated, while `$params`
  * is an array representing the additional parameters supplied in the validation rule.
  *
+ * In version 2.0.11 additional argument `$validator` containing reference to related `InlineValidator` was added.
```

Comment:
IMO line 16 must be modified as follows:

```
function foo($attribute, $params, $validator)
```

This line should be changed to:

```
Parameter `$validator` refers to the related [[InlineValidator]] object and is available since version 2.0.11
```

All the following lines of PHP could be removed